test/test_gels.cc

```diff
@@ -18,6 +18,7 @@
 #include <cstdio>
 #include <cstdlib>
 #include <utility>
+#include <typeinfo>
```

Comment:
Is this used?

Reply:
Thanks for pointing this out, I was using this when I was working to figure out the solution for the `if` check to see if it would be `cholqr` or `Geqrf`. The output was giving me 0 or 1, and when I would compare that to an actual integer it would throw an error. Which I later found out was due to the type being related to the `enum` after using `<typeinfo>`.  Have gone ahead and deleted it.